clans/src/main/java/me/mykindos/betterpvp/clans/clans/listeners/ClansMovementListener.java

```diff
@@ -146,33 +166,12 @@ public void onClanStuckTeleport(ClanStuckTeleportEvent event) {
             return;
         }
 
-
         ClanRelation relation = clanManager.getRelation(clanManager.getClanByPlayer(player).orElse(null), territoryOptional.get());
 
         if (relation == ClanRelation.ENEMY) {
-            event.setDelayInSeconds(3 * 5);
+            event.setDelayInSeconds(120);
         } else {
-            event.setDelayInSeconds(2 * 5);
+            event.setDelayInSeconds(60);
         }
     }
-
-    @EventHandler
-    public void onSpawnTeleport(SpawnTeleportEvent event) {
-        if (event.isCancelled()) return;
-
-        gamerManager.getObject(event.getPlayer().getUniqueId()).ifPresent(gamer -> {
-            if (gamer.getClient().hasRank(Rank.ADMIN)) {
-                return;
-            }
-
-            if (clanManager.getClanByLocation(event.getPlayer().getLocation()).isPresent()) {
-                UtilMessage.message(event.getPlayer(), "Spawn", "You can only teleport to spawn from the wilderness.");
-                event.setCancelled(true);
-            } else {
-                event.setDelayInSeconds(30);
-            }
-
-        });
-
-    }
```

Comment:
Honestly leave this in, the spawn command can just be disabled for players via config in-case we want to re-enable in the future